source/leuven/genius_quick_start.rst

```diff
@@ -71,9 +71,18 @@ walltimes of 3 days or less.
 
 Submit to a compute node
 ~~~~~~~~~~~~~~~~~~~~~~~~
-To submit to a compute node it all boils down to specifying the required number of nodes and cores. As the nodes have a single user policy we recommend to always request all available cores per node (36 cores in this case). For example to request 2 nodes with each 36 cores you can submit like this::
-
-   $ qsub -l nodes=2:ppn=36  -l walltime=2:00:00  -A myproject  myjobscript.pbs
+Submitting a compute job boils down to specifying the required number of nodes, cores-per-node, memory and walltime.
+The nodes on the ``pbs``, ``bigmem`` and ``gpu`` partitions have ``SHARED`` policy (but the nodes on the ``amd`` partition have ``SINGLEUSER`` policy).
```

Comment:
It looks like you're talking about Moab's "node allocation policy",
which is how Moab is deciding which node(s) to allocate for a given job.
The policy employed on Genius seems to be the "MINRESOURCE" one,
where indeed eligible small jobs are preferentially sent to nodes which
are already partially being used by other small jobs.

I think it would be best if you would rephrase this accordingly, and so avoiding
terms like "SHARED" and "SINGLEUSER" and writing "node allocation policy"
instead of "node policy". This would reduce the confusion with what is called
"node access policy" (which is a related topic, but not one that needs to be
addressed in this Quickstart page).

Also, so far I couldn't find out whether the node allocation policy is 
different on the AMD partition compared to the other partitions.
As far as I can see, it's "MINRESOURCE" everywhere.

Reply:
On AMD nodes, we have still the singleuser policy:
```
$ checknode r28i39n01 | grep -i policy
EffNodeAccessPolicy: SINGLEUSER
```
and as Mag correctly demonstrated, only cascadelake nodes have shared policy.
Jan suggests to keep the current MOAB setting as it is, and only change everything consistently when we move to Slurm.
I will revise the text accordingly.

Reply:
and b.t.w. where do you see the `MINRESOURCE` phrase? I have not seen this before.
Can you plz give an example how to retrieve this?

Reply:
Please keep in mind that "node allocation policy" and "node access policy"
are two different (though related) things ;).

The node allocation policy is the more simple of the two, and you can e.g.
find documentation about it here (including the `MINRESOURCE` mode):
https://docs.adaptivecomputing.com/maui/5.2nodeallocation.php

The node access policy is a more complex one. Strictly speaking I would call this
a property of a job  (not of a node or partition, although node- or partition-
specific settings are also influential here). This is because (at least on Genius)
it can also depend on the job resources (i.e. how many cores and how much memory)
what the job's node access policy will be (i.e. "shared" or "singlejob").
As e.g. Mag pointed out, there are essentially two different parts of Genius in
this regard: 
* one part where the jobs' node access policy is always "singlejob"
(the skylake "batch" and "bigmem" nodes, as well as the AMD nodes)
* another part where the same policy is applied, unless the amount of
cores and memory is within certain limits (then the "shared" node access
policy is given). This applies to all cascadelake nodes and all GPU nodes,
as well as Superdome.

(disclaimer: not sure if even this is a complete explanation, it could be even
more subtle)

Reply:
And when updating this quickstart page, keep in mind that there also this other page,
(perhaps you can refer to it, or put some info there instead of in the quickstart page):

https://docs.vscentrum.be/en/latest/jobs/specifying_resources.html#enforcing-the-node-specification

Reply:
To be honest, I am more confused than before reading this PR (mostly because I thought the situation was much simpler). If I now understand things correctly, submitting a job with:
```
qsub -l nodes=1:ppn=4:skylake
```
will always lead to a situation where no other jobs can run on the node where my job runs? In that case, isn't it better to recommend users to request all cores, so they automatically get all the memory on the node and have more options with respect to pinning/binding? In both cases they will be _blocking_ an entire node.

Of course the different behavior of skylake and cascadelake complicates things a lot, so for simplicity towards the user, Ehsan's proposed text is fine for me. Also note that this will become irrelevant in a few months, when we switch to Slurm on genius.

Reply:
I tired to 
- avoid using the ``SHARED`` and ``SINGLEUSER`` jargons, but similar to the other documentation page, I mention nodes access policy being 'shared' or 'singleuser' based on what users can retrieve from `checkjob` (although that is tricky)
- made a simple distinction between skylake vs cascadelake nodes regarding the node access policy, with basic `qsub` examples
- I will include an additional sentence how to enforce getting skylake nodes using the architecture as a feature (in response to Steven's comment below).

Reply:
It's definitely looking better now! But considering the Tier-2 audience, I think it would
be better to present this in a more gradual way, by first giving the basic info and only
give the more advanced info in a second step.

The contents of the basic part would include:
* an example for requesting two full nodes (the one that's already in there)
* recommending that "request only as much resources as needed by their applications"
  and there include e.g. your 8-core example
* to **not** yet mention the ":skylake"/"-l feature=skylake" (or cascadelake) stuff
  (and also not the node access policy etc). Just mentioning that the above example jobs 
  may end up running on either a Skylake or a Cascadelake node. 

The contents of the more advanced part would include:
* mentioning that users can control the node type with those ":skylake"/"-l feature=skylake"
  (and cascadelake) flags
* explaining the difference in the Skylake and Cascadelake compute nodes when
  it comes to node access policy. Here you can then mention that for small (few-core) jobs,
  it is best to either not select a specific nodetype feature or to select the Cascadelake feature.
* mentioning that there are also AMD compute nodes, but that those are described in a separate
  section below. If you want you can then also describe the node access policy in that AMD
  section.